tools/sgfirebaseconfig/tools.go

```diff
@@ -0,0 +1,236 @@
+package sgfirebaseconfig
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+type CSP struct {
+	Rules          map[string]Group
+	DefaultRules   []string
+	Provided       []string
+	ReportToDomain string
+}
+
+type Group struct {
+	connectSrc []string
+	fontSrc    []string
+	frameSrc   []string
+	imgSrc     []string
+	scriptSrc  []string
+	styleSrc   []string
+	workerSrc  []string
+}
+
+func (csp *CSP) rules() { // Default list of things needed for a Saga app.
```

Comment:
All this rules are very Einride specific. Sage is open source and I don't really see how this fits here, or do I miss something?

Reply:
Rewrite this now. Broke out the `AddRule` functionality so that we can add anything company specific in the projects sage config. Have a look at the updated PR comment above.